packages/opencensus-exporter-zipkin/src/zipkin.ts

```diff
@@ -14,12 +14,21 @@
  * limitations under the License.
  */
 
-import {Exporter, ExporterBuffer, ExporterConfig, RootSpan, Span} from '@opencensus/core';
+import * as coreTypes from '@opencensus/core';
+import {Exporter, ExporterBuffer, ExporterConfig, RootSpan, Span, SpanKind} from '@opencensus/core';
 import {logger, Logger} from '@opencensus/core';
-import {prototype} from 'events';
 import * as http from 'http';
 import * as url from 'url';
 
+const STATUS_CODE = 'census.status_code';
+const STATUS_DESCRIPTION = 'census.status_description';
+
+const messageEventTypeTranslation: {[k: number]: string} = {
```

Comment:
Nit: could you name this MESSAGE_EVENT_TYPE_TRANSLATION?

Reply:
Done.